### PR TITLE
Hideout: heating 2 - added 3 crickent

### PR DESCRIPTION
**Pull request by ckaram91**
[Discord Post: #bugs-data](https://discord.com/channels/668387296973684737/668387739808563221/920322391337500682)

image
image

They added 3 crickent as a requirement for Heating L2.

### CODE REVIEW

File: hideout.json [hunk 1]
```diff
@@ -600,6 +600,12 @@
                     "name": "5e2af2bc86f7746d3f3c33fc",
                     "quantity": 2,
                     "id": 233
+                },
+                {
+                    "type": "item",
+                    "name": "56742c284bdc2d98058b456d",
+                    "quantity": 3,
+                    "id": 311
```
Comment (ckaram91) [inline hideout.json:608]: Is this `id` just supposed to be unique on the page?

Comment (thaddeus) [reply hideout.json:608]: Correct!

Comment (thaddeus) [reply hideout.json:608]: There is a CLI tool if you have node installed, which can find the next available ID. If not, you'll need to manually look for the next ID. The CLI tool can be called with `node ./cli.js help`

Comment (ckaram91) [reply hideout.json:608]: Ah cool, thanks! Yeah, that's what I did: just kept looking for the highest existing id (manual binary search via notepad find).

**Review by thaddeus — APPROVED** [review]
LGTM 👍 Thanks for the update!

**Pull request merged**